### DISCUSSION TITLE
Outbound Transfers Module

### DISCUSSION THREAD

**Discussion by GSLevel** [DSJAS/DSJAS#54] [Show and tell]
To sum it up, this module allows you to transfer funds to a bank account that you don't own (It really just transfers it to a ghost account, to be exact it transfers to account 999,999, which doesn't exist since no one's going to make that many accounts), or to a zelle email. You can also use discord webhooks to log them (inspired by discussion #8 ).

Hopefully this will be helpful and happy scambating!

Repo: https://github.com/GSLevel/outbound-transfers

**Comment by ejv2** [comment]
Great! Wonderful work. I will have a tinker with it myself. Although I don't use Discord myself, it would be perfectly possible for me to set up a trivial server on my host that logs all incoming connections and returns success which could stand in for a Discord logger.

As I see it now, this module is heavily reliant upon the structure of the default theme. There is no problem with this, but it would be ideal to work with any theme we wish. I will have a play with it and maybe open a PR on your repo if I manage to make it theme independent.

Thanks a lot for this. It looks fantastic and is a great example of how to use modules.

**Reply by GSLevel** [reply]
Sounds good, thanks for the feedback.
